next.fonts.ts

```diff
@@ -0,0 +1,29 @@
+import { Source_Sans_3, Open_Sans, IBM_Plex_Mono } from 'next/font/google';
```

Comment:
what do you thinks about putting that in `util/font` ?

Reply:
Nah, because it's not really an util